### PR TITLE
Adding sorting to VMI list

### PR DESCRIPTION
**Pull request by metalice**
Signed-off-by: Matan Schatzman <mschatzm@redhat.com>

## 📝 Description

> Added sorting for VMI list table headers.

## 🎥 Demo

image

### CODE REVIEW

File: src/views/virtualmachinesinstance/list/VirtualMachinesInstancesRow.tsx [hunk 1]
```diff
@@ -28,7 +30,8 @@ const VirtualMachinesInstancesRow: React.FC<VirtualMachinesInstancesRowProps> =
         <VMStatusConditionLabelList conditions={obj?.status?.conditions?.filter((c) => c.reason)} />
       </TableData>
       <TableData id="created" activeColumnIDs={activeColumnIDs}>
-        {obj?.metadata?.creationTimestamp}
+        <GlobeAmericasIcon />{' '}
+        {format(new Date(obj?.metadata?.creationTimestamp), 'MMM dd, yyyy, h:mm a')}
```
Comment (avivtur) [inline src/views/virtualmachinesinstance/list/VirtualMachinesInstancesRow.tsx:34]: nice :) 
can you please add a tooltip that contains UTC time format as we have in 4.10?

Comment (metalice) [reply src/views/virtualmachinesinstance/list/VirtualMachinesInstancesRow.tsx:34]: I don't see it in vmi list. It's there and I'm missing it?

Comment (yaacov) [reply src/views/virtualmachinesinstance/list/VirtualMachinesInstancesRow.tsx:34]: we will replace this all this components (in all the lists) with a common component from the component repo, no need for this one to be perfect

**Pull request merged**